File: 🟢 Easy/1903 - Largest Odd Number in String/_1903LargestOddNumberinString.cpp
class Solution {
public:
    string largestOddNumber(string num) {
        int i = num.length() - 1;
        int to = 0;
        while (i >= 0) {
            if ((num.at(i) - '0') % 2 != 0) {
                to = i + 1;
                break;
            }
            i--;
        }

        return num.substr(0, to);
    }
};

File: 🟢 Easy/1903 - Largest Odd Number in String/_1903LargestOddNumberinString.py
class Solution:
    def largestOddNumber(self, num: str) -> str:
        i = len(num) - 1
        to = 0
        while i >= 0:
            if (int(num[i]) % 2) != 0:
                to = i + 1
                break

            i -= 1

        return num[0:to]